go.mod

```diff
@@ -5,152 +5,22 @@ go 1.17
 replace (
 	github.com/kudobuilder/kuttl => github.com/layer5io/kuttl v0.4.1-0.20200806180306-b7e46afd657f
 	vbom.ml/util => github.com/fvbommel/util v0.0.0-20180919145318-efcd4e0f9787
-// golang.org/x/sys => golang.org/x/sys v0.0.0-20200826173525-f9321e4c35a6
+//	golang.org/x/sys => golang.org/x/sys v0.0.0-20220319134239-a9b59b0215f8
+	github.com/Sirupsen/logrus => github.com/sirupsen/logrus v1.8.1
+	k8s.io/api => k8s.io/api v0.22.8
+	k8s.io/apimachinery => k8s.io/apimachinery v0.22.8
+	k8s.io/cli-runtime => k8s.io/cli-runtime v0.22.8
+	k8s.io/client-go => k8s.io/client-go v0.22.8
+	k8s.io/kubectl => k8s.io/kubectl v0.22.8
 )
 
 require (
 	github.com/go-logr/logr v0.4.0
```

Comment:
This needs to be: `github.com/go-logr/logr v1.2.3`

Reply:
Thank you for this, @jbyers19